uvm/api/com/untangle/uvm/util/StringUtil.java

```diff
@@ -127,4 +127,23 @@ public static String longToHumanReadable(long value){
         }
         return humanValue;
     }
+
+    /**
+     * Returns whether input string is either null or empty i.e. length = 0
+     * @param input Input to validate
+     * @return
+     */
+    public static boolean isEmpty(String input) {
+        return input == null || input.length() == 0;
+    }
+
+
+    /**
```

Comment:
No, let's not do this.  Just have the caller properly handle the negative logic.

Reply:
Okay, will remove isNotEmpty method